docs/03-node-operators/04-setup/03-configuration/03-reference.md

```diff
@@ -347,6 +348,57 @@ scoring {
 Check out [Configure RSKj node for mining](/node-operators/merged-mining/configure-mining)
 for detailed information about the `miner` configuration.
 
+## miner.stableGasPrice
+
+The `stableGasPrice` feature allows miners to set a predictable gas price for transactions, establishing a minimum rate that is less affected by market volatility and better aligned with fiat currency values like USD or EUR. This enhances cost management and improves overall transaction efficiency.
+
+The source configuration supports two methods: ETH_CALL and HTTP_GET.
+
+- **ETH_CALL**: This method retrieves the gas price directly from a specified smart contract, allowing for real-time adjustments based on the contract's logic.
+- **HTTP_GET**: This method fetches gas price data from an external API, providing flexibility to pull market data from trusted sources.
+
+**Example of ETH_CALL**
+```plaintext
+stableGasPrice {
+    enabled = true
+
+    source {
+        method = "ETH_CALL"
+        params {
+            from = "0x0000000000000000000000000000000000000000"
+            to = "0xCA41630b09048b6f576459C34341cAB8Dc2cBE7E"
+            data = "0xdf16e52d"
+        }
+    }
+
+    minStableGasPrice = 476190000000 # Minimum gas price
+    refreshRate = 6 hours          # How often to check for updates
+}
+```
```

Comment:
The data inside params and after "ETH_CALL" doesn't need a comma?

Reply:
Not exactly! The configuration I used when setting up the node for testing follows that format. It doesn't require a comma in that context

Reply:
Ok. Cool